### PR TITLE
CNV-4363 Delete references to CNV technology preview

### PR DESCRIPTION
**Pull request by lmandavi**
This PR is associated with https://issues.redhat.com/browse/CNV-4363. All references to CNV technology preview have been removed in assemblies as part of this PR. Because this is purely deletion of references, this does not require SME review. Ready for peer review. As per @aburdenthehand 's guidance, this PR needs to be cherry picked to 4.5 and 4.6.

### CODE REVIEW

File: virt/install/installing-virt.adoc [hunk 1]
```diff
@@ -15,7 +15,6 @@ to subscribe to and deploy the {VirtProductName} Operators.
 * {product-title} {product-version}
 
 :FeatureName: {VirtProductName}
```
Comment (vikram-redhat) [inline virt/install/installing-virt.adoc:17]: Should this also remove the FeatureName attribute? @aburdenthehand ?

Same below in other places.

Comment (aburdenthehand) [reply virt/install/installing-virt.adoc:17]: Right you are. Thanks for the catch.
#23978 takes care of it

**Pull request merged**